tests/library/inspector/inspectorTest.ts

```diff
@@ -186,9 +186,12 @@ class Recorder {
     await this.page.mouse.move(box.x + box.width / 2, box.y + box.height / 2);
   }
 
-  async trustedClick() {
-    await this.page.mouse.down();
-    await this.page.mouse.up();
+  async trustedClick(options?: {
+    button?: 'left' | 'right' | 'middle';
+    clickCount?: number;
```

Comment:
let's drop clickCount as it is unused

Reply:
https://github.com/microsoft/playwright/pull/31634/commits/3ece2f5aea8d745cd69a914d59d84117f8c49fdc